packages/build-tools/src/docProcessor.mjs

```diff
@@ -0,0 +1,104 @@
+import { Logger } from "@aurodesignsystem/auro-library/scripts/utils/logger.mjs";
```

Comment:
You'll also want to add an `export` reference in `packages/build-tools/package.json`:

```
"exports": {
    ".": "./src/index.js",
    "./formVersionWriter": "./src/formVersionWriter.mjs",
    "./docProcessor": "./src/docProcessor.mjs"
  }
```